packages/scandipwa/src/component/FieldSelect/FieldSelect.container.js

```diff
@@ -236,6 +246,30 @@ export class FieldSelectContainer extends PureComponent {
         }
     }
 
+    handleOpenDirection() {
+        const windowHeight = document.documentElement.clientHeight;
+        const rect = this.fieldRef.getBoundingClientRect();
+        const bottomPosition = Math.round(windowHeight - rect.bottom);
+        const minimunDistance = 200;
```

Comment:
Add please this as constant and make more descriptive name

Reply:
Done!